README.md

```diff
@@ -142,6 +142,12 @@ Default: `google.maps`
 
 In case you want to provide your own Google Maps object, pass it in as googleMaps. The default is the global google maps object.
 
+#### useNominatim
+Type: `Boolean`
+Default: `false`
+
+Set to true if you prefer to use Open Street Maps Nominatim service instead of google APIs.  Note that usage with autolookup fields is prohibited by OSM terms of serivce so a 'Search' button will be added to the component to query for suggests.  This option also removes the google maps api page include requirement.
```

Comment:
As an API it would be more independent to have something like `geocodeProvider: 'nominatim'`. That would be scalable for other implemenations, too.